app.js

```diff
@@ -12,14 +14,146 @@ function toggleMobileMenu() {
+    div.setAttribute('id', `${i}`);
+    div.innerHTML = `
+   <div class="card-left-mobile">
+   <img src="${portfolioItemsDetail[i].images}" alt="snap-card">
+   </div>
+   <div class="card-right">
+     <h2>${portfolioItemsDetail[i].name}</h2>
+     <ul class="cano">
+       <li>${portfolioItemsDetail[i].title}<i class="fa fa-circle"></i></li>
+       <li>${portfolioItemsDetail[i].stack}<i class="fa fa-circle"></i></li>
+       <li>2015</li>
+     </ul>
+     <p>${portfolioItemsDetail[i].text}</p>
+     <ul class="lang">
+       ${portfolioItemsDetail[i].workStack
+    .map((stack) => `<li>${stack}</li>`)
+    .join('')}
+     </ul>
+     <button class="btn-default portfolio-items-detail">See project</button>
+   </div>
+   <div class="card-left-desk">
+     <img src="${portfolioItemsDetail[i].images}" alt="snap-card">
+   </div>
+   `;
+    portfolio.appendChild(div);
+  }
+}
+const portfolioItems = document.querySelectorAll('.portfolio-items-detail');
+if (portfolioItems) {
+  portfolioItems.forEach((cards) => {
+    cards.addEventListener('click', (e) => {
+      const idValue = e.target.closest('.card').id;
+      for (let i = 0; i < portfolioItemsDetail.length; i += 1) {
+        if (portfolioItemsDetail[i].id === idValue) {
+          const popupPage = document.createElement('page');
+          popupPage.classList.add('popup-page');
+          // console.log(porfolioItemsDetail[i].name);
+          popupPage.innerHTML = `
+          <div class="background"></div>
+          <div class="card flex-main">
+            <button class="close-btn" type="submit">
+              <i class="fas fa-times"></i>
+            </button>
+            <div class="card-right">
+              <h2>${portfolioItemsDetail[i].name}</h2>
+              <ul class="cano">
+                <li>${portfolioItemsDetail[i].title}<i class="fa fa-circle"></i></li>
+                <li>${portfolioItemsDetail[i].stack}<i class="fa fa-circle"></i></li>
+                <li>${portfolioItemsDetail[i].year}</li>
+              </ul>
+              <div class="snap-image">
+                <img src="${portfolioItemsDetail[i].images}" alt="${portfolioItemsDetail[i].name}">
+              </div>
+              <div class="flex-nl">
+                <p>${portfolioItemsDetail[i].text}</p>  
+                <ul class="lang">
+                ${portfolioItemsDetail[i].workStack.map((stack) => `<li>${stack}</li>`).join('')}
+                </ul>
+                  <div class="link-button">
+                    <button class="btn-default"><a href="${portfolioItemsDetail[i].live} class="see-btn">See live<img
+                    src="images/live.png" alt="live-link" /></a></button>
+                    <button class="btn-default"><a href="${portfolioItemsDetail[i].source} class="see-btn">See source<img
+                    src="images/Vector.png" alt="live-link" /></a></button>
+                  </div>
+              </div>
+            </div>
+          </div>
+          `;
```

Comment:
- [ ] As mentioned by the last CR:

> You are missing the title of the project and the x button which is to close the popup. 👍

- [ ] Please 🙏🏽 add the close button on the desktop version to be able to close the modal page for a better UX.
- [ ] When I open the modal on the mobile version I cannot see the complete information of the modal please make sure that the modal is centered on the screen for a better UX.
- [ ] Please 🙏🏽 correct the styles to be similar to the Figma template. As mentioned by the last CR  Your popup currently does not match the provided figma template on desktop devices as shown from the images below.

| Figma | Your design |
|---|---|
| ![image](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/58642949/ba501f11-5600-4099-b899-a524acd6c535) | ![image](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/58642949/360957c7-a3d8-4a80-bd71-89e51f8a4b68) |
| ![image](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/58642949/1579b652-cc5f-40d6-af74-7b0cd15f4b13) | ![image](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/58642949/c9d37d72-add6-4ee6-87ec-000f54c99b07) |